src/development/LocalTest/Services/LocalApp/Implementation/LocalAppFile.cs

```diff
@@ -0,0 +1,87 @@
+#nullable enable
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+
+using Altinn.Platform.Storage.Interface.Models;
+
+using LocalTest.Configuration;
+using LocalTest.Services.LocalApp.Interface;
+using LocalTest.Services.Localtest.Interface;
+
+namespace LocalTest.Services.LocalApp.Implementation
+{
+    public class LocalAppFile : ILocalApp
+    {
+        private readonly LocalPlatformSettings _localPlatformSettings;
+        public LocalAppFile(IOptions<LocalPlatformSettings> localPlatformSettings)
+        {
+            _localPlatformSettings = localPlatformSettings.Value;
+        }
+
+        public async Task<string?> GetXACMLPolicy(string appId)
+        {
+            return await File.ReadAllTextAsync(Path.Join(GetAppPath(appId), $"App/config/authorization/policy.xml"));
+        }
+
+        public async Task<Application?> GetApplicationMetadata(string appId)
+        {
+            var filename = Path.Join(GetAppPath(appId), $"App/config/applicationmetadata.json");
+            if (File.Exists(filename))
+            {
+                var content = await File.ReadAllTextAsync(filename, Encoding.UTF8);
+                return JsonConvert.DeserializeObject<Application>(content);
+            }
+            
+            return null;
+        }
+
+        public async Task<Dictionary<string, Application>> GetApplications()
+        {
+            var ret = new Dictionary<string, Application>();
```

Comment:
could be renamed to something more descriptive e.g. appDictionary

Reply:
I can't recall any discussion about naming of return variables, and I'm sure we are inconsistent. 

I personally prefer to name after content and not "purpose". Robert C. Martin seems to agree with me (and Stephanie) while Martin Fowler seems to agree with Ivar.